src/components/ServiceDetails/AboutService/Overview.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { withStyles } from "@material-ui/styles";
+
+import { useStyles } from "./styles";
+import Tags from "./Tags";
+
+const Overview = ({ classes, description, service_url, tags }) => {
```

Comment:
Shall we rename this to `ServiceOverview`?

Reply:
removed [it](https://github.com/singnet/refactored-dapp/pull/13/commits/ff31558b08c2d932f1e21bad426a4a7b98372297)

Reply:
[renamed](https://github.com/singnet/refactored-dapp/pull/13/commits/2f2490bb23ada9ea79a97918bc61a2b1ac234364)